src/Stryker.Core/Stryker.Core.UnitTest/MutationTest/MutationTestProcessTests.cs

```diff
@@ -100,7 +98,11 @@ public void ShouldCallMutantOrchestratorAndReporter()
             orchestratorMock.Setup(x => x.GetLatestMutantBatch()).Returns(mockMutants);
             orchestratorMock.Setup(x => x.Mutate(It.IsAny<SyntaxNode>())).Returns(CSharpSyntaxTree.ParseText(SourceFile).GetRoot());
             orchestratorMock.SetupAllProperties();
-            var options = new StrykerOptions(devMode: true, excludedMutations: new string[] { }, fileSystem: new MockFileSystem());
+            var options = new StrykerOptions()
+            {
+                DevMode = true,
+                ExcludedMutations = new Mutator[] { }
```

Comment:
Also Mutation?

Reply:
Mutators apply mutations (and a specific mutation is a mutant). From a user perspective I changed the option from mutator to mutation because for a user it's more logical to say I don't want this mutation to happen instead of I don't want this mutator enabled. But internally the option disables or enables the mutators. Does that make sense?

Reply:
It does - and also explains the whole idea behind this renaming. I think this translation of the terms should happen on the domain border to avoid this kind of confusing assignments - but it's maybe beyond this PR.